workflow-templates/im-build-db-ci.yml

```diff
@@ -31,7 +31,10 @@ env:
   DB_SERVER_NAME: 'localhost'
   DB_SERVER_PORT: '1433'
   DEFAULT_BRANCH: 'main' # TODO: verify default branch name
+  ARTIFACTORY_URL: 'https://artifacts.mktp.io/artifactory/api/nuget/nuget-all' # TODO: This is used when grabbing dependency scripts and when publishing dependency scripts for others to use. Remove if that isn't something your repo needs to do.
   ARTIFACTORY_RETRIEVAL_URL: 'https://artifacts.mktp.io/artifactory/api/nuget/nuget-all' # TODO: This can be removed if your database doesn't have any dependency objects stored in artifactory
+  FLYWAY_MANAGED_SCHEMAS: 'dbo' # TODO: update this comma separated value with all of the schemas that Flyway manages. E.g. dbo,MyCustomSchema,AnotherSchema
+  SQL_LOGIN_USERNAME: 'SA' # The default SA account SQL Server comes with
```

Comment:
Is this something that someone would normally pull out of a secret or is this generally what you would use?